src/summarycode/plugin_consolidate.py

```diff
@@ -338,6 +338,9 @@ def get_holders_consolidated_components(codebase):
                 if license_expression:
                     child.extra_data['normalized_license_expression'] = license_expression
                     child.save(codebase)
+                if "unknown" in license_expression:
```

Comment:
The `license_expression` is added to both `normalized_license_expression` and `normalized_unknown_license_expression`. If `unknown` add only to `normalized_unknown_license_expression` and not in `normalized_license_expression`.